hosting.md

```diff
@@ -14,12 +14,17 @@ Here are some convenient options for serving the HTML:
 serving static content.
 
 The archive script generates index pages in markdown that are
-designed for a Jekyll build, so you can do something like this:
+designed for a Jekyll build, so you can do something like this
+from within this directory (where you checked out code):
 
 ```
-jekyll serve --incremental
+gem install jekyll bundler
+jekyll serve
```

Comment:
Why is the `--incremental` flag gone?

Reply:
I just wanted the docs to reflect the way I actually tested it.  The original comment didn't explain why you'd use `--incremental` or what the implications are, so I just wanted to keep it simple.  Feel free to submit a PR that fleshes out the Jekyll section a bit here.  You can either stack on top of mine or wait for Tim to merge a few of these commits.

Reply:
I assume it's because the jekyll server is being run persistently, and it will update itself automatically whenever the archive is being updated (also) incrementally by `archive.py`.

Reply:
It's still an experimental feature, according to Jekyll.  I assume it works fine for the simple Zulip use case, but I really want to keep the Jekyll docs simple, with just enough pointers to get up and running.

https://jekyllrb.com/docs/configuration/incremental-regeneration/

Reply:
But eventually people will have to encounter the case when the archive has to be updated. There needs to be a documentation on how this is handled with just `jekyll serve`, without the experimental feature.

Reply:
`jekyll serve` already looks for updates; it just re-builds the whole archive instead of being smart about it.

If you are interested in making the Jekyll docs better, it's probably a good use of your time to just write up some detailed docs in a readme file and submit a PR.   There's other stuff to explain, like how we make the front matter, etc.

We also want to make sure our solution isn't overly Jekyll-based, since I think a lot of folks would prefer that we just built static HTML that they could serve up directly.

Reply:
I updated the commit so that it mentions `--incremental` now.

Reply:
For a Jekyll site that just contains a Zulip archive, I don't think the flag makes much difference, because the "last updated" timestamps force most/all of the pages to be rebuilt each time. In the original leanprover setup, the archive was part of a larger Jekyll site and made up 95% of the Jekyll build time. The `--incremental` flag saved time when we were editing the rest of the site, not so much when we updated the archive.